front-end/src/components/helpers/ExampleTranscript.js

```diff
@@ -0,0 +1,13 @@
+export const exampleTranscript = {"data": [
+        {
+            "project_id": "Your unique project id",
+            "trace_type": "speak",
+            "trace_payload": "{\"type\":\"speak\",\"payload\":{" +
+                "\"type\":\"message\":\"<voice name=\\\"AnyVoiceName\\\">Insert chatbot message.</voice>\"}}"
+        },
+        {
+            "project_id": "Your unique project id",
+            "trace_type": "intent",
+            "trace_payload": "{\"type\":\"intent\",\"ELICIT\":false,\"payload\":{" +
+                "\"query\":\"unique query\",\"intent\":{\"name\":\"unique intent\"},\"confidence\":1}}"
```

Comment:
I dont think the intent has to be unique. Also, I think it would be more clear if it was formatted like: <unique intent> or [UNIQUE INTENT] just to make it clear what the user is supposed to change (this goes for all fillable fields). Also, I don't think we need a confidence field @helias22 can confirm.

Reply:
No we don't need confidence. 
It's also better to just have speak and text as trace_types. 
Because even if they do have intents, the formatter will trim it out.